aten/src/ATen/native/quantized/cpu/tensor_operators.cpp

```diff
@@ -0,0 +1,57 @@
+#include <ATen/ATen.h>
+#include <ATen/ExpandUtils.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+namespace native {
+
+/*
+All comparator operators will be named "<aten op name>_quantized_cpu".
+'_out' will be appended for the 'out' variant of the op.
+
+TODO: This is an inefficient implementation that uses `.dequantize`.
+      Need a more efficient implementation.
+*/
```

Comment:
For scalar comparisons, can you quantize the scalar (much less expensive than dequantizing a tensor)?

Reply:
A more efficient solution will be part of the upcoming PRs.